com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalLitSubTarget.cs

```diff
@@ -605,25 +605,29 @@ static class LitRequiredFields
         {
             public static readonly FieldCollection Forward = new FieldCollection()
             {
-                StructFields.Attributes.uv1,                            // needed for meta vertex position
+                StructFields.Attributes.uv1,
+                StructFields.Attributes.uv2,
                 StructFields.Varyings.positionWS,
                 StructFields.Varyings.normalWS,
                 StructFields.Varyings.tangentWS,                        // needed for vertex lighting
                 StructFields.Varyings.viewDirectionWS,
-                UniversalStructFields.Varyings.lightmapUV,
+                UniversalStructFields.Varyings.staticLightmapUV,
+                UniversalStructFields.Varyings.dynamicLightmapUV,
```

Comment:
I have concerns here regarding the shadergraph as they do not handle conditional interpolator packing that well. So it might result more interpolators even with disabled varyings. I think we need mobile test pass to be sure on that